DEVELOPMENT_GUIDE.md

```diff
@@ -152,8 +154,13 @@ The script automatically updates hardcoded ARN partitions to match the output pa
 python3 bin/add_transform_test.py --template-file template.yaml --disable-update-partition
 ```
 
-Note that please always check the generated output is as expected. This tool does not guarantee correct output.
+Please always check the generated output is as expected. This tool does not guarantee correct output.
+
+#### Transform failures
+
+To test a SAM template results in a specific transform error, add the SAM template under [`tests/translator/input`](https://github.com/aws/serverless-application-model/tree/develop/tests/translator/input), and a JSON file with the expected `errorMessage` as a top-level value under [`tests/translator/output`](https://github.com/aws/serverless-application-model/tree/develop/tests/translator/output). The JSON file must be prefixed with `error_` and include the same basename as the input SAM template (e.g. `my_cool_template.yaml` for input, and `error_my_cool_template.json` for the expected error).
```

Comment:
Are you sure the template file doesn't need to be `error_my_cool_template.yaml`? Example: https://github.com/aws/serverless-application-model/blob/develop/tests/translator/input/error_api_invalid_auth.yaml

Reply:
Good catch, fixed.